recipes/newrelic/apm/dotNet/linux-systemd.yml

```diff
@@ -18,6 +18,26 @@ keywords:
 
 processMatch: []
 
+preInstall:
+  requireAtDiscovery:
+    - |
+      rm -rf /tmp/dotnet-introspector >/dev/null
+      curl -s https://virtuoso-testing.s3-us-west-2.amazonaws.com/dotnet-is-linux-x64.gz -o /tmp/dotnet-is-linux-x64.gz >/dev/null
+      mkdir -p /tmp/dotnet-introspector >/dev/null
+      zcat /tmp/dotnet-is-linux-x64.gz > /tmp/dotnet-introspector/nri-lsi-dotnet
+      rm -f /tmp/dotnet-is-linux-x64.gz >/dev/null
+      chmod -R 777 /tmp/dotnet-introspector >/dev/null
```

Comment:
Can we not 777?  If you need to make sure it is executable maybe just a 'chmod u+x', that will give the current user execution permission for the file.

Reply:
Updated with the suggested change.